<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingCreatePage</class>
 <widget class="QWidget" name="TrainingCreatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>473</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#backButton {
	background: transparent;
	border: none;
	font-family: Open Sans;
	font-size: 24px;
	line-height: 34px;
	font-weight: 600;
	letter spacing: 0%
}

#statusPanel {
	background-color: #EEF4F6;
	border: 1px solid #34343426;
	border-radius: 8px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="trainingCreatePageLayout">
   <property name="spacing">
    <number>24</number>
   </property>
   <property name="leftMargin">
    <number>14</number>
   </property>
   <property name="topMargin">
    <number>20</number>
   </property>
   <property name="rightMargin">
    <number>14</number>
   </property>
   <property name="bottomMargin">
    <number>20</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="backButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>34</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>208</width>
         <height>34</height>
        </size>
       </property>
       <property name="text">
        <string>Create Training</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Resources/Resources.qrc">
         <normaloff>:/images/nav_back.svg</normaloff>:/images/nav_back.svg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="startTrainingButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: white;
	border: 2px solid #1A1A1A;
	border-radius: 8px;
	font-family: Open Sans;
	font-weight: 500;
	font-size: 18px;
	color: #1A1A1A;
}

QPushButton:hover {
    background-color: #D6ECFF;
}

QPushButton:pressed {
    background-color: #B7DCFF;
    padding-top: 2px;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Start Training</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>152</width>
         <height>42</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
	background: #FDB515;
	border: none;
	border-radius: 8px;
	font-family: Open-Sans;
	font-weight: 500;
	font-size: 18px;
	color: white;
}

QPushButton:hover {
    background-color: #E6A512;
}

QPushButton:pressed {
    background-color: #CC8F0F;
    padding-top: 2px;
	padding-left: 2px;
}</string>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="container">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>1399</width>
        <height>618</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="containerLayout">
       <property name="spacing">
        <number>18</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="TrainingCreatePanel" name="trainingPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>867</width>
           <height>618</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="statusPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>514</width>
           <height>618</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>514</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TrainingCreatePanel</class>
   <extends>QWidget</extends>
   <header>trainingcreatepanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
